repair/learn/learn.py

```diff
@@ -88,9 +92,7 @@ def __init__(self, env, feat_info, output_dim, bias=False):
         self.model = TiedLinear(self.env, feat_info, output_dim, bias)
         self.featurizer_weights = {}
 
-    def fit_model(self, X_train, Y_train, mask_train):
-        n_examples, n_classes, n_features = X_train.shape
-
+    def fit_model(self, training_data):
```

Comment:
Can you add a docstring here specifying that `training_data` is a `torch.utils.data.Dataset`?